### DISCUSSION TITLE
Remoteplay-button not available after 3.0.2 for non GPU-users?

### DISCUSSION THREAD

**Discussion by RoligKanin** [redphx/better-xcloud#214] [General]
With 3.0.2 and the remote-play button moved to "jump back in", I cant see it anymore.

Not currently subscribed to Gamepass Ultimate, so I suspect the whole "Jump back in" section is not visible and naturally I cant see the "remote play" button either. Am I the only one?

Had to revert back to 3.0.1 to keep the remoteplay functionality.

**Comment by redphx** [comment]
Free users don't have the "Jump back in" section?

**Reply by RoligKanin** [reply]
Doesnt look like it. I have the same sections as if i wasnt logged in (apart from "play with firends").

Mine is in norwegian, so roughly translated i have:
"Play with friends" (only when logged in).
"Most popular in the cloud"
"Newly added"
"Going away soon"
"Genres"
"All games"

Tried launching Fortnite, to see if "jump back in" section would pop up, but it didnt.

**Comment by redphx** [comment]
Fixed in 3.0.2.1

**Reply by RoligKanin** [reply]
Perfect <3
